Content.Shared/Movement/Systems/SharedMoverController.cs

```diff
@@ -443,6 +445,14 @@ private bool IsAroundCollider(SharedPhysicsSystem broadPhaseSystem, TransformCom
                 sound = moverModifier.FootstepSoundCollection;
                 return true;
             }
+            
+            // If got the component in yml and no shoes = no sound. Delta V
+            if (_entities.TryGetComponent(uid, out NoShoesSilentFootstepsComponent? _) &
+                !_inventory.TryGetSlotEntity(uid, "shoes", out var _))
+            {
+                return false;
+            }
+            // Delta V NoShoesSilentFootsteps till here.
```

Comment:
![image](https://github.com/DeltaV-Station/Delta-v/assets/16548818/8cb775d8-f827-45e6-b358-7ae7e157f727)
I'm actually going to request that this be made using a pre-existing component. You can simply make a footstep collection for Felinids that contains only a completely silent .ogg file, and set Felinids to use this collection for their footsteps. Since it gets overridden by shoes, this would make them walk silently while barefoot without needing to touch any upstream code at all.

Reply:
could it not be changed to be optional too, a silent ogg file is 1. probably some performance landmine 2. hacky as fuck

Reply:
>performance impact
Literally zero. It already plays a .ogg clientside whenever a felinid walks. It is a default component on all humanoids.

We don't need to touch upstream code at all if this can be done with a single line of yaml in a non-upstream file. Even if you consider it "Hacky", bear in mind everything we do is similarly, if not more "hacky". I consider a single-line change in a non c# file to be the most elegant solution of all.

Reply:
> Literally zero. It already plays a .ogg clientside whenever a felinid walks. It is a default component on all humanoids.

It is still doing stuff every step it doesn't have to

Reply:
I've also considered that it could be done with a felinidcomponent that remcomps footstepmodifiercomponent on init. 

Another potential option is tagsystem.removetag footstep , but both options I havent investigated deep enough into what would happen if either were outright removed. I happen to know that every shoe in the game temporarily modifies the species footstepmodifiercomponent, so removing it might not be safe. Whereas giving them a silent footstep sound file has a net zero performance change, and is guaranteed to work for this purpose.